src/SearchResults.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import SearchTableRow from "./SearchTableRow";
+import CustomerProfile from "./CustomerProfile";
+const SearchResults = ({ results }) => {
+  console.log(results);
+  const [selectedProfile, setSelectedProfile] = useState("");
+
+  const selectId = customerId => {
+    setSelectedProfile(customerId);
+  };
+  return (
+    <>
+      <table className="table">
+        <thead>
+          <tr>
+            <th scope="col">ID</th>
+            <th scope="col">Title</th>
+            <th scope="col">First name</th>
+            <th scope="col">Surname</th>
+            <th scope="col">E-mail</th>
+            <th scope="col">Room No</th>
+            <th scope="col">Check-in Date</th>
+            <th scope="col">Check-out Date</th>
+            <th scope="col">Numer of Nights</th>
+            <th scope="col">Profile</th>
+          </tr>
+        </thead>
+        <tbody>
+          {props.results.map((booking, index) => {
```

Comment:
If you run the app with `npm run start`, and then open [Chrome developer tools ](https://developer.chrome.com/docs/devtools/open/), then go to the console tab you'll see it complaining that `props` isn't defined.

This is because you've already destructured `{results}` on line 4.

So you can just say `{results.map...` here.